asv/util.py

```diff
@@ -995,6 +995,13 @@ def get_cpu_info():
     elif sys.platform.startswith('darwin'):
         sysctl = which('sysctl')
         return check_output([sysctl, '-n', 'machdep.cpu.brand_string']).strip()
+    elif sys.platform.startswith('win'):
+        try:
+            from win32com.client import GetObject
+            cimv = GetObject("winmgmts:root\cimv2")
```

Comment:
`\c` seems a little unsafe - probably best to use `\\c` or `r"winmgmts:root\cimv2"`

Reply:
@casperdcl windows docs seem to suggest this is a way to go:
https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-tasks--processes, why `\\` would be safer?

Reply:
looks like an escape code otherwise. (e.g. `\n` -> newline)

Reply:
Got it, thanks, was not aware of that. I will prepare PR for that.